Plots/MapProjections/NCL_native_2.py

```diff
@@ -0,0 +1,91 @@
+"""
+NCL_native_2_lg.py
+================
+
+This script illustrates the following concepts:
+   - Drawing filled contours over a mercator map
+   - Overlaying contours on a map without having lat,lon coordinates
+   - Drawing a map using the medium resolution map outlines
+   - Turning on map tickmark labels with degree symbols
+   - Selecting a different color map
+   - Zooming in on a particular area on a mercator map
+   - Using best practices when choosing plot color scheme to accomodate visual impairments
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/native_2.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/native_2_lg.png
+"""
+
+###############################################################################
+
+# Import packages:
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mticker
+import cartopy.feature as cfeature
+
+import geocat.datafiles as gdf
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and
+# load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/1994_256_FSD.nc"), decode_times=False)
+t = ds.FSD.isel(time=0)
+
+###############################################################################
+# Plot:
+
+# Generate figure (set its size (width, height) in inches)
+fig = plt.figure(figsize=(10, 10))
+
+# Generate axes using Cartopy and draw coastlines
+ax = plt.axes(projection=ccrs.Mercator())
+ax.coastlines(linewidths=0.5)
+ax.add_feature(cfeature.LAND, facecolor="lightgray")
+
+# Set extent to include latitudes from 34 to 52 and longitudes from 128
+# to 144
+ax.set_extent([128, 144, 34, 52], ccrs.PlateCarree())
+
+# Contourf-plot data (for filled contours)
+pt = t.plot.contourf(
+    ax=ax,
+    transform=ccrs.PlateCarree(),
+    vmin=0,
+    vmax=70,
+    levels=15,
+    cmap="inferno",
+    cbar_kwargs={
+        "extendrect": True,
+        "orientation": "vertical",
+        "ticks": np.arange(0, 71, 5),
+        "label": ""})
+
+# Draw gridlines
+gl = ax.gridlines(crs=ccrs.PlateCarree(), draw_labels=True,
+    dms=False,
+    x_inline=False,
+    y_inline=False,
+    linewidth=1,
+    color="k",
+    alpha=0.5)
```

Comment:
This is purely my preference, but I think more transparent grid lines are less distracting. Try an alpha of 0.25 and see what you think of it.